### PR TITLE
WFCORE-752 - add test for multiple deployment overlay

### PR DESCRIPTION
**Pull request by hsvabek**
Test for multiple deployment overlays
https://bugzilla.redhat.com/show_bug.cgi?id=1229711
https://issues.jboss.org/browse/WFCORE-752

### CODE REVIEW

**Comment by stuartwdouglas** [comment]
Retest this please

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/cli/DeploymentOverlayCLITestCase.java [hunk 1]
```diff
@@ -210,7 +210,7 @@ public static void after() throws Exception {
         ear2.delete();
         FileUtils.deleteDirectory(ear2_exploded);
         replacedLibrary.delete();
-        replacedAjsp.delete();
+//        replacedAjsp.delete();
```
Comment (stuartwdouglas) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/cli/DeploymentOverlayCLITestCase.java]: why has this been commented out?

Comment (hsvabek) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/cli/DeploymentOverlayCLITestCase.java]: Methods @BeforeClass and @AfterClass are invoke twice (at least on my local machine).
replacedAjsp - this file is in target location... other files are generated - it is reason why it works after comment.

I am trying to find out where is the problem.

Comment (bstansberry) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/cli/DeploymentOverlayCLITestCase.java]: This is fine.

**Comment by bstansberry** [comment]
retest this please

**Pull request merged**